storage-operator/deploy/crds/storage_v1alpha1_volume_crd.yaml

```diff
@@ -67,6 +67,56 @@ spec:
           - storageClassName
           type: object
         status:
+          properties:
+            failedContext:
+              properties:
+                errorCode:
+                  enum:
+                  - 0
+                  - 1
+                  - 2
+                  - 3
+                  format: int64
+                  type: integer
+                errorMessage:
+                  type: string
+              type: object
+            pendingContext:
+              properties:
+                jobId:
+                  description: JID of the Salt job in progress.
+                  type: string
+                size:
```

Comment:
Why should the `size` be stored in the `status` of a `Volume`? This seems redundant because it's a field in the `spec` of the corresponding `PersistentVolume` once created.

Reply:
Yes, it's a bit redundant.
I'll explain why I did that and maybe we can figure out a better way :slightly_smiling_face: 

My goal here was to make the `size` returned by the Salt call persistent, that way if the operator crash or the PV creation fails (with some transient error) and I have to reschedule I can reuse the size from the annotation instead of re-doing the Salt call each time.

Reply:
As seen with you, I'll remove it.